README.md

```diff
@@ -37,7 +37,7 @@ make docker-push
 
 ## Register as a Fybrik module
 
-To register HWM as a Fybrik module apply `hello-world-module.yaml` to the fybrik-system namespace of your cluster.
+To register HWM(Hello World Module) as a Fybrik module apply `hello-world-module.yaml` to the fybrik-system namespace of your cluster.
```

Comment:
please add space between "HWM" and "(Hello World Module)"

Reply:
Done thanks